### DISCUSSION TITLE
Use `:where()` in `tailwindcss-typography`?

### DISCUSSION THREAD

**Discussion by stefanprobst** [tailwindlabs/tailwindcss#5286] [Ideas]
currently, when using `tailwindcss-typography` to style MDX content with custom components, it's a bit annoying that most of the time the `prose` styles will have higher specificity.

does it make sense to wrap `.prose` in `:where()`? ([browser support](https://caniuse.com/mdn-css_selectors_where) is actually pretty good)

```html
<style>
  :where(.prose) p { color: lime; }
  .paragraph { color: hotpink; }
</style>
<article class="prose">
  <p class="paragraph">Lorem ipsum.</p>
</article>
```

**Comment by eboye** [comment]
This would absolutely makes sense to me!

**Comment by stefanprobst** [comment]
looks like it might be happening: https://github.com/tailwindlabs/tailwindcss-typography/pull/203

**Comment by adamwathan** [comment]
Ha just saw this now — I dumped this idea in my Trello board at the end of July!

![image](https://user-images.githubusercontent.com/4323180/134811391-5adc4a1d-e1a2-4334-a667-b991801f50d0.png)

Definitely a good idea, hopefully can make it work without introducing any new issues 💪
